sound/soc/sof/topology.c

```diff
@@ -2340,13 +2322,6 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 	return ret;
 }
 
-static int sof_route_unload(struct snd_soc_component *scomp,
```

Comment:
@plbossart also can we please leave this on for now. I need this for my free_topology() fix changes. I will submit the PR by tomorrow.

Reply:
@plbossart this change is good. we can remove it.

Reply:
done